src/www/model/errors.ts

```diff
@@ -101,7 +101,7 @@ export class VpnStartFailure extends RedFlagNativeError {}
 
 // This must be kept in sync with:
 //  - cordova-plugin-outline/apple/src/OutlineVpn.swift#ErrorCode
-//  - cordova-plugin-outline/apple/vpn/PacketTunnelProvider.h#NS_ENUM
+//  - cordova/apple/OutlineAppleLib/Sources/VPNSources/PacketTunnelProvider.h#NS_ENUM
```

Comment:
I think this changed?
```suggestion
//  - cordova/apple/OutlineAppleLib/Sources/OutlineTunnelSources/PacketTunnelProvider.h#NS_ENUM
```